lib/quby/answers/services/updates_answers.rb

```diff
@@ -12,30 +12,27 @@ def initialize(answer)
         end
 
         def update(new_attributes = {})
-          answer.raw_params = new_attributes
-
-          attribute_filter   = FiltersAnswerValue.new(answer.questionnaire)
-          attribute_filter.filter(new_attributes).each do |name, value|
-            answer.send("#{name}=", value)
-          end
+          save_raw_params_on_error(new_attributes) do
+            attribute_filter = FiltersAnswerValue.new(answer.questionnaire)
+            attribute_filter.filter(new_attributes).each { |name, value| answer.send("#{name}=", value) }
 
-          answer.extend AnswerValidations
-          answer.cleanup_input
-          answer.validate_answers
+            answer.extend AnswerValidations
+            answer.cleanup_input
+            answer.validate_answers
 
-          if answer.errors.empty?
-            if new_attributes["rendered_at"]
-              started_at = Time.at(new_attributes["rendered_at"].to_i)
+            if answer.errors.empty?
+              if new_attributes["rendered_at"]
+                started_at = Time.at(new_attributes["rendered_at"].to_i)
```

Comment:
Ik bedenk mij nu dat het beter met een `.present?` check kan, `''.to_i` is immers 0.